src/Microsoft.AspNetCore.SignalR/IHubActivator.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.SignalR
+{
+    public interface IHubActivator
+    {
+        HubWrapper<THub, TClient> Create<THub, TClient>(IServiceProvider provider)
```

Comment:
It might be a little confusing that `Create` here creates a HubWrapper and not a Hub directly. Maybe think about renaming.